src/main/webapp/app/overview/course-exercises/course-exercises.component.ts

```diff
@@ -22,6 +22,14 @@ export class CourseExercisesComponent implements OnInit, OnDestroy {
     public weeklyIndexKeys: string[];
     public weeklyExercisesGrouped: object;
 
+    public upcomingExercises: Exercise[];
+
+    public numberOfQuizExercises = 0;
```

Comment:
Can you use a map instead with key=ExerciseType and value=number? Then please use a for loop and iterate over the exercise types so that in the future (in case we add a new exercise type), this functionality is still working correctly